test/types/records/init/copy/initEqualsErrorMessage.good

```diff
@@ -0,0 +1,6 @@
+initEqualsErrorMessage.chpl:3: In function 'test':
+initEqualsErrorMessage.chpl:4: error: could not find initializer for type 'r' accepting expression of type 'int(64)'
```

Comment:
I'd probably rephrase line 2's error message as:

```
could not find a copy initializer ('init=') for type 'r' from type 'int(64)'
```

And then I might remove the note that follows (since the person writing the code may not be the person who wrote type 'r' in which case they  may not actually be able to do this and it could just be frustrating; if they did write'r', hopefully that will be an obvious thing they could consider doing and the mention of 'init=' above would give that hint.  Lines 4–6 look good to me.

But I think it'd be worth getting additional opinions other than mine on this as well.

Reply:
Thanks, your error message is much better than what I came up with.